src/test/resources/features/Pillar2SubmissionSetUpPage.feature

```diff
@@ -1,4 +1,4 @@
-@p2tests
+@ignore
```

Comment:
why we are ignoring this test

Reply:
These steps were added during Pillar 2 Submission Acceptance tests setup and we dont need them anymore.